pkg/cmd/container/create.go

```diff
@@ -357,8 +365,49 @@ func generateRootfsOpts(args []string, id string, ensured *imgutil.EnsuredImage,
 			// error message is from Podman
 			return nil, nil, errors.New("no command or entrypoint provided, and no CMD or ENTRYPOINT from image")
 		}
+
+		entrypointPath = processArgs[0]
+
 		opts = append(opts, oci.WithProcessArgs(processArgs...))
 	}
+
+	systemdPaths := []string{
+		"/usr/sbin/init",
+		"/sbin/init",
```

Comment:
Wondering  if `/sbin` should have higher priority over `/usr/sbin`

Reply:
Yeah, `/sbin/init` is more common. I'll move higher